asapdiscovery-docking/asapdiscovery/docking/openeye.py

```diff
@@ -184,8 +197,18 @@ def _dock(
                     }
                     posed_ligand.set_SD_data(sd_data)
 
+                    # Generate info about which target was actually used by multi-reference docking
+                    if isinstance(set, DockingInputMultiStructure):
+                        docked_target = set.complexes[result.GetReceptorIndex()]
+                        print(f"{docked_target.target.target_name} was docked")
```

Comment:
Remove print iff working. 

Also we may want to consider seperating the code paths rather than using `isinstance` but is okay for now.

Reply:
Yeah I see why a different code path vs `isinstance` would be better but i couldn't figure out how to do that reasonably without copying many many lines of code